runTest1.sh

```diff
@@ -2,26 +2,50 @@
 
 apk add -U wget ca-certificates
 
-# Testing for version 0.2.x of the container
-
 # Download data
-wget "https://drive.google.com/uc?export=download&id=0B7S2ZMhdzWwbT3AtdDk0S3dJaDg" -O best_fit_fluxes.csv
+BEST_FIT_FLUXES_IN_06=best_fit_fluxes_06.csv
```

Comment:
Currently tests are checking for 0.2 and 0.6 iso2flux versions. Could you please @cfoguet add additional logic on develop on this file to test for 0.7 iso2flux best fit fluxes file if they are different to 0.6? Thanks!